### PR TITLE
FEAT: edb.configuration.export operations

### PR DESCRIPTION
**Pull request by ring630**

### CODE REVIEW

**Review by svandenb-dev — APPROVED** [review]
LGTM

**Review by svandenb-dev — APPROVED** [review]
LGTM

File: src/pyedb/configuration/cfg_operations.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        self.custom_extent_units = kwargs.get("custom_extent_units", None)
+        self.include_partial_instances = kwargs.get("include_partial_instances", None)
+        self.keep_voids = kwargs.get("keep_voids", None)
+        self.check_terminals = kwargs.get("check_terminals", None)
+        self.include_pingroups = kwargs.get("include_pingroups", None)
+        self.expansion_factor = kwargs.get("expansion_factor", None)
+        self.maximum_iterations = kwargs.get("maximum_iterations", None)
+        self.preserve_components_with_model = kwargs.get("preserve_components_with_model", None)
+        self.simple_pad_check = kwargs.get("simple_pad_check", None)
+        self.keep_lines_as_path = kwargs.get("keep_lines_as_path", None)
```
Comment (SMoraisAnsys) [inline src/pyedb/configuration/cfg_operations.py:51]:
```suggestion
        self.signal_list = kwargs.get("signal_list")
        self.reference_list = kwargs.get("reference_list")
        self.extent_type = kwargs.get("extent_type")
        self.expansion_size = kwargs.get("expansion_size")
        self.use_round_corner = kwargs.get("use_round_corner")
        self.output_aedb_path = kwargs.get("output_aedb_path")
        self.open_cutout_at_end = kwargs.get("open_cutout_at_end")
        self.use_pyaedt_cutout = kwargs.get("use_pyaedt_cutout")
        self.number_of_threads = kwargs.get("number_of_threads")
        self.use_pyaedt_extent_computing = kwargs.get("use_pyaedt_extent_computing")
        self.extent_defeature = kwargs.get("extent_defeature")
        self.remove_single_pin_components = kwargs.get("remove_single_pin_components")
        self.custom_extent = kwargs.get("custom_extent")
        self.custom_extent_units = kwargs.get("custom_extent_units")
        self.include_partial_instances = kwargs.get("include_partial_instances")
        self.keep_voids = kwargs.get("keep_voids")
        self.check_terminals = kwargs.get("check_terminals")
        self.include_pingroups = kwargs.get("include_pingroups")
        self.expansion_factor = kwargs.get("expansion_factor")
        self.maximum_iterations = kwargs.get("maximum_iterations")
        self.preserve_components_with_model = kwargs.get("preserve_components_with_model")
        self.simple_pad_check = kwargs.get("simple_pad_check")
        self.keep_lines_as_path = kwargs.get("keep_lines_as_path")
```
Dictionnary `get` command returns `None` by default. No need to specify it.

**Review by SMoraisAnsys — CHANGES_REQUESTED** [review]
I request minor changes, LGTM otherwise !

**Review by SMoraisAnsys — APPROVED** [review]

**Pull request merged**